src/components/Annotation/Annotation.js

```diff
@@ -0,0 +1,123 @@
+// @flow
+import * as React from 'react';
+import classNames from 'classnames';
+import { FormattedMessage } from 'react-intl';
+import getProp from 'lodash/get';
+import noop from 'lodash/noop';
+
+import Avatar from 'box-react-ui/lib/components/avatar';
+import { ReadableTime } from 'box-react-ui/lib/components/time';
+import Tooltip from 'box-react-ui/lib/components/tooltip';
+
+import CommentText from './CommentText';
+import InlineDelete from './InlineDelete';
+import CommentInlineError from './CommentInlineError';
+import UserLink from './UserLink';
+import messages from './messages';
+
+import './Annotation.scss';
+
+const ONE_HOUR_MS = 3600000; // 60 * 60 * 1000
+
+type Props = {
+    id: string,
+    message: string,
+    created_by: User,
+    permissions: AnnotationPermissions,
+    created_at: string,
+    modified_at?: string,
+    onDelete?: Function,
+    isPending?: boolean,
+    error?: ActionItemError
+};
+
+type State = {
+    isFocused?: boolean
+};
+
+class Annotation extends React.Component<Props, State> {
+    state = {
+        isFocused: false
+    };
+
+    onKeyDown = (event: SyntheticKeyboardEvent<>): void => {
+        const { nativeEvent } = event;
+        nativeEvent.stopImmediatePropagation();
+    };
+
+    handleCommentFocus = (): void => {
+        this.setState({ isFocused: true });
+    };
+
+    handleCommentBlur = (): void => {
+        this.setState({ isFocused: false });
+    };
+
+    render() {
+        const {
+            id,
+            created_at,
+            created_by,
+            permissions,
+            message,
+            isPending = false,
+            error,
+            onDelete = noop
+        } = this.props;
+        const { isFocused } = this.state;
+        const canDelete = getProp(permissions, 'can_delete', false);
+        const createdAtTimestamp = new Date(created_at).getTime();
+
+        return (
+            <div className='ba-annotation-container'>
+                <div
+                    className={classNames('ba-annotation', {
+                        'ba-is-pending': isPending || error,
+                        'ba-is-focused': isFocused
+                    })}
+                    onBlur={this.handleCommentBlur}
+                    onFocus={this.handleCommentFocus}
+                >
+                    <Avatar className='ba-annotation-avatar' {...created_by} />
+                    <div className='ba-annotation-content'>
```

Comment:
Are there any divs here that we don't style with positioning? If there are could we remove them and style the sub components in a pre-fixed way instead?